src/Pages/ResultList/Component/ResultListContents.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const ResultListContents = () => {
+  return RESTAURANT_LIST.map(data => (
+    <li className="resultListContentsLi" key={data.id}>
+      <figure className="restaurantItem">
+        <Link to={data.detailUrl} className="contentLink">
+          <div className="thumb">
+            <img
+              className="resultImage"
+              alt={data.address}
+              src={data.imageUrl}
+            />
+            {data.dayOff === 'true' && (
```

Comment:
```suggestion
            {data.dayOff && (
```
mock 데이터에 해당 값은 boolean 값으로 되어야 합니다. 
현재는 'true' 라는 string으로 지정 되어있는데 true or false 라는 boolean으로 지정 되면
위와 같이 수정할 수 있습니다.